shared/fs/header/header-container.js

```diff
@@ -11,26 +11,23 @@ type OwnProps = {
 }
 
 type DispatchProps = {
-  onOpenBreadcrumb: (path: string) => (evt?: SyntheticEvent<>) => void,
+  _onOpenBreadcrumb: (path: string, evt?: SyntheticEvent<>) => void,
   _onOpenBreadcrumbDropdown: (
     dropdownItems: Array<Types.PathBreadcrumbItem>,
-    isTeamPath: boolean,
-    onOpenBreadcrumb: (path: string) => (evt?: SyntheticEvent<>) => void
+    isTeamPath: boolean
   ) => (evt?: SyntheticEvent<>) => void,
 }
 
 const mapStateToProps = () => ({})
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  onOpenBreadcrumb: (path: string) => (evt?: SyntheticEvent<>) => {
+  _onOpenBreadcrumb: (path: string, evt?: SyntheticEvent<>) => {
     dispatch(navigateTo([fsTab, {props: {path: Types.stringToPath(path)}, selected: 'folder'}]))
     evt && evt.stopPropagation()
   },
-  _onOpenBreadcrumbDropdown: (
-    dropdownItems: Array<Types.PathBreadcrumbItem>,
-    isTeamPath: boolean,
-    onOpenBreadcrumb: (path: string) => (evt?: SyntheticEvent<>) => void
-  ) => (evt?: SyntheticEvent<>) =>
+  _onOpenBreadcrumbDropdown: (dropdownItems: Array<Types.PathBreadcrumbItem>, isTeamPath: boolean) => (
```

Comment:
the second arrow is incorrect right

Reply:
Right. I'll fix. I also missed fixing the same pattern in the sortbar, so I'll do that too.